README.md

```diff
@@ -12,19 +12,20 @@ $ npm install spid-passport
 
 ## Utilizzo
 ### Configurazione
-Sono necessari i parametri di configurazione per l'Identity Provider e per il Service Provider e nello specifico il costruttore prende in input due oggetti e una callback di verifica:
+Sono necessari i parametri di configurazione per l'Identity Provider e per i diversi Service Provider e nello specifico il costruttore prende in input due oggetti e una callback di verifica:
 
 ##### Service Provider: 
-- (String) `entity_id` - Nome entita che fornisce il servizio
-- (Stirng) `private_key` - Chiave privata del Service Provider (Formato PEM)
-- (String) `certificate` - Certificato Service Provider (Formato PEM)
-- (String) `assert_endpoint` - Endpoint sul quale ricevere la response dall'identity provider
+- (String) `issuer` - Nome entita che fornisce il servizio
```

Comment:
non è chiaro se è una stringa o una URL (come nell'esempio sottostante), magari inseriamo in questo elenco due parole sul formato atteso per ogni parametro